backend/test/MyRoom_test.ts

```diff
@@ -0,0 +1,31 @@
+import assert from "assert";
+import { ColyseusTestServer, boot } from "@colyseus/testing";
+
+// import your "arena.config.ts" file here.
+import appConfig from "../src/arena.config";
+import { MyRoomState } from "../src/rooms/schema/MyRoomState";
+
+describe("testing your Colyseus app", () => {
```

Comment:
This is strange test, as it looks like it is testing just Colyseus behavior and not custom functionality, which should be priority of tests.

Reply:
Good point, I went ahead and removed that entire directory (it came packaged with the template)